docs/ops/movement/Unique_10.md

```diff
@@ -0,0 +1,119 @@
+
+**Attributes**:
+
+* *axis*
+
+  * **Description**: *axis* specifies the input tensor's axis along which the subtensors are processed. When the axis is set to the maximum value of the underlying integer type it means that the input tensor should be processed elementwise (the operator then looks for unique individual numbers in the whole flattened tensor).
+  * **Range of values**: The accepted range of values is `[-R, R-1]` where `R` is the rank of the input tensor.
+  * **Type**: int
+  * **Default value**: MAX_INT
+  * **Required**: *no*
+
+* *sorted*
+
+  * **Description**: controls whether the unique elements in the output tensor are sorted in ascending order.
+  * **Range of values**:
+    * false - output tensor's elements are not sorted
+    * true - output tensor's element are sorted
+  * **Type**: boolean
+  * **Default value**: true
+  * **Required**: *no*
+
+* *index_element_type*
+
+  * **Description**: controls the data type of the output tensors containing indices.
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i64"
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: A tensor of type *T* and arbitrary shape. **Required.**
+* **2**: A tensor of type *T_AXIS*. The allowed tensor shape is 1D with a single element or a scalar. **Optional**
+  When provided this input is used to "divide" the input tensor into slices along the specified axis before unique elements processing starts. When this input is not provided the operator works on a flattened version of the input tensor (elementwise processing).
+
+**Outputs**
+
+* **1**: The output tensor containing unique elements (individual values or subtensors). This tensor's type matches the type of the first input tensor: *T*. The values in this tensor are either sorted ascendingly or maintain the same order as in the input tensor. The shape of this output depends on the values of the input tensor and will very often be dynamic. Please refer to the article describing how [Dynamic Shapes](https://docs.openvino.ai/latest/openvino_docs_OV_UG_DynamicShapes.html) are handled in OpenVINO.
+* **2**: The output tensor containing indices of the locations of unique elements. The indices map the elements in the first output tensor to their locations in the input tensor. The index always points to the first occurence of a given unique output element in the input tensor. This is a 1D tensor with type *T_IND*.
+* **3**: The output tensor containing indices of the locations of elements of the input tensor in the first output tensor. This means that for each element of the input tensor this output will point to the unique value in the first output tensor of this operator. This is a 1D tensor with type *T_IND*.
+* **4**: The output tensor containing the number of occurences of each unique value produced by this operator in the first output tensor. This is a 1D tensor with type `int64`.
```

Comment:
What type does PyTorch return for this output? Could you please check?

Reply:
Sure, I've checked in manually in python because this information is missing in the specs https://pytorch.org/docs/stable/generated/torch.unique.html. Anyways the returned data type is also `int64`:
```
>>> torch.unique(t, return_counts=True)[1].dtype
torch.int64
```

Reply:
What do you think about an attribute to set the output type for the 4th output? 
Similar as *index_element_type* for indices (output 3).

Reply:
> What do you think about an attribute to set the output type for the 4th output? Similar as _index_element_type_ for indices (output 3).

This is something I've considered as well, it's in the last bullet point in the PR's description.  Overall I don't see the need to configure the data type for this particular output, I think it should rather be an unsigned data type but ONNX requires `int64` - that's why I decided to go for this element type in OV. I'm open to suggestions though.